atc/db/team.go

```diff
@@ -563,18 +563,6 @@ func savePipeline(
 		return 0, false, err
 	}
 
-	_, err = psql.Update("resources").
-		Set("resource_config_id", nil).
-		Where(sq.Eq{
-			"pipeline_id": pipelineID,
-			"active":      false,
```

Comment:
why is this block removed? Is it conflicting with the changes?

Reply:
The functionality is incorporated into the updated `saveResource` query: https://github.com/concourse/concourse/pull/8002/files#diff-bc775ddab7d4f0909d112312360cad5826a0958c9b5e83ce16911337ff8ecd48R1249